backup-parameter-store/build.sbt

```diff
@@ -5,14 +5,12 @@ organization := "com.gu"
 
 description:= "Backs up parameter store properties. To be executed on an interval"
 
-version := "1.0"
```

Comment:
There is no need to specify the `sbt-riffraff-artifact` version in `build.sbt`. We are bumping this to version `1.1.18` in this PR in `backup-parameter-store/project/plugins.sbt`